gym/wrappers/normalize_action.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+import gym
+from gym import spaces
+
+
+class NormalizeAction(gym.ActionWrapper):
+    r"""Rescale the continuous action space of the environment from [-1, 1]. """
```

Comment:
Should the wrapper allow rescaling to other values than just `-1, 1`?

Reply:
@hartikainen Thanks a lot for the advice, now it is supported for arbitrary range scaling.